docs/user.md

```diff
@@ -1,284 +1,286 @@
-
-##### Alpha
-
-By default, the alpha mode is 'OPAQUE' and set to 'BLEND', if 
-- the BaseColorFactor or DiffuseFactor alpha value is less than 1.0  
-- the Alpha channel from the BaseColor or Diffuse image texture is connected.  
-
-Please note, that separate alpha maps are currently not specified in glTF 2.0 and so not working.
-The alpha channel has to be linked from the above defined image textures. Following picture shows the correct and simple usage for the glTF Metallic Roughness node:
-
-![glTF Material Node Alpha Blend](glTF_Material_Node_Alpha_Blend.png)
-
-To use 'MASK' for blending, the AlphaMode has to be set from 0.0 to 1.0. In this case, as specified by glTF 2.0, the AlphaCutoff value is used and exported.
-
-##### Double sided
-
-In glTF 2.0, the information if a mesh has to be rendered double sided or not, is stored in the material and not the mesh. By default, double sided is disabled and if a glTF 2.0 scene is exported, the back faces are culled. If a mesh should be renderered without back face culling, enable the double sided material by setting the slider to 1.0.
-
-##### Vertex color usage
-
-By default, `Use COLOR_0` is disabled and the slider has to be set to 1.0 for enabling it.
-
-Furthermore, an `Attribute` input has to be linked to `COLOR_0` for a correct export.
-
-In the `Name` property field of the `Attribute` node, the string of the `Vertex Colors` has to be entered. In the following picture, the name is `COLOR_0` but any name can be used.
-
-![glTF Material Node Alpha Blend](glTF_Material_Node_Color.png)
-
-#### PBR Metallic Roughness
-
-This is the complete Metallic Roughness material node.
-
-![glTF Metallic Roughness Node](glTF_Metallic_Roughness_Node.png)
-
-#### PBR Specular Glossiness
-
-The specular glossiness material is not part of core glTF 2.0. The material is defined by the extension `KHR_materials_pbrSpecularGlossiness`.
-
-![glTF Specular Glossiness Node](glTF_Specular_Glossiness_Node.png)
-
----
-
-### External Tools
-
-The following section describes several tools, how they can optimal be used with the Blender glTF 2.0 exporter. 
-
-#### Substance Painter
-[Substance Painter](https://www.allegorithmic.com/products/substance-painter) is a 3D painting software allowing you to texture, render and share your work.  
-At point of writing, the exporter of Substance Painter does not have a preset for glTF 2.0 Metallic Roughness export. However, this can be easily configured:  
-
-![glTF SubstancePainter](glTF_SubstancePainter.png)
-
-Important is the order of `occlusion`: red channel, `roughness`: green chanel, `metallic`: blue channel. Even the glTF 2.0 property is named `metallicRoughnessTexture`, the above channel order assignment is the only valid one.
-
-Furthermore, please make sure, that the `normal` is exported for `OpenGL`. Even your render engine is using a different graphics API, this is the only valid one.
+User Documentation
+------------------
+
+### glTF 2.0 Materials
```

Comment:
Not sure why GitHub thinks this entire document has changed.  Did the line-endings change, or the encoding change?  It's hiding the actual diffs here.

Reply:
yeah I see that and been afraid of breaking something, but apparently not, and I've no idea of what changes its about.
Like you said, encoding maybe ? I'm on windows 10, and notepad++ open this file with "UTF-8 (without BOM)" encoding.